js/index.js

```diff
@@ -1,7 +1,36 @@
 // Iteration 1: Names and Input
+let hacker1 = "Fernando"
+console.log("The driver's name is " + hacker1)
 
+let hacker2 = "Alejandro"
+console.log("The navigator's name is " + hacker2)
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log("The driver has the longest name, it has " + hacker2.length + " characters.")
+}else if (hacker1.length < hacker2.length){
+    console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters.")
+} else {
+    console.log("Wow, you both have equally long names, XX characters!.")
```

Comment:
Hola Fer, buena interpolación en esta iteración aquí podríamos agregar la longitud de cualquier hacker ya que aqui ambos tienen la misma o podríamos agregar Template literals

```suggestion
    console.log("Wow, you both have equally long names, `${hacker1.length}` characters!.")
```